LiteFarmOrg/LiteFarm
# Lf 2659/multi language support for bulk sensor upload format on the front end

To test:

1. open the sensor upload modal. 
2. add the following attached files and check the validation is passing successfully.
3. Also, try editing the same files to see the validation error. 

[test-sensors-multilanguage.zip](https://github.com/LiteFarmOrg/LiteFarm/files/9194077/test-sensors-multilanguage.zip)

For more details, please check: 
https://lite-farm.atlassian.net/browse/LF-2659
